week-2-wide-perspective/w02e05-simple-server-side/src/logger.ts

```diff
@@ -0,0 +1,19 @@
+import { createLogger, format, transports } from 'winston'
+
+export const logger = createLogger({
+    level: 'error',
+    format: format.combine(
+        format.label({
+            label: 'Error'
+        }),
+        format.timestamp(),
+        format.prettyPrint()
+    ),
+    transports: [
+        new transports.File({ filename: `${process.cwd()}/src/logs/error.log` })
+    ]
+})
+
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
Tego bym nie dawał. 
Z praktycznego pkt. widzenia, logi w konsoli na produkcji i tak się przydają. Jak wystawiasz proda za pomocą np. `pm2` to i tak on zarządza messages, które masz w terminalu. To po prostu wygodne.

Zrobił bym tak, żeby na produkcji "Poza" konsolą dorzucać plik - tak na "wrazie czgo". to się przyda.

Natomiast w `development` tylko konsola.